dspace/modules/journal-submit/src/main/java/org/datadryad/submission/DryadGmailService.java

```diff
@@ -46,7 +46,7 @@
      * @throws IOException
      */
     public DryadGmailService(File credentialPath, String clientsecretsFile, String scope, String user) {
-        myUserID = user;
```

Comment:
This change does not work on dev.... it's working as user on dev, presumably working as user on production. When it's changed to "me" on dev, I get an authorization exception.